[16.0][REF][ADD] report_csv, report_format_option (#864)
OCA/reporting-engine: report_format_option/readme/CONFIGURE.rst

In case the exported report should be encoded in another system than UTF-8, following
fields of the report record (*Settings > Technical > Reports*) should be populated accordingly.

* Encoding: set an encoding system (such as cp932)
* Encode Error Handling: select 'Ignore' or 'Replace' as necessary.
  * 'Ignore': in case of an encoding error, the problematic character will be removed from the exported file.
  * 'Replace': in case of an encoding error, the problematic character will be replaced with '?' symbol.
  * Leaving the field blank: in case of an encoding error, the report generation fails with an error message.

> * Line Ending: Select the type of line ending—'CRLF', 'CR' or leave blank, as necessary.
>   * 'CRLF': Stands for 'Carriage Return' + 'Line Feed'. It's used as a line ending in Windows, signifying the end of a line and the start of a new one.
>   * 'CR': Stands for 'Carriage Return'. It's used in classic Mac OS as a line ending, moving the cursor to the beginning of the line.
>   * Leaving this field blank will default to using 'LF' (Line Feed), which is the standard line ending in Unix/Linux systems, denoting the end of a line.

Reviewer: Let's make it less verbose (and more maintainable).

```suggestion
* Line Ending: Select the type of line ending, 'CRLF' or 'CR', as necessary.
  * 'CRLF': 'Carriage Return' + 'Line Feed' (Windows)
  * 'CR': 'Carriage Return' (classic Mac OS)
  * Leaving this field blank defaults to using 'LF' (Line Feed), the default behavior of Odoo.
```
